ipc/provider/src/manager/subnet.rs

```diff
@@ -141,4 +142,11 @@ pub trait TopDownCheckpointQuery: Send + Sync {
     ) -> Result<Vec<CrossMsg>>;
     /// Get the block hash
     async fn get_block_hash(&self, height: ChainEpoch) -> Result<Vec<u8>>;
+    /// Get the validator change set from start to end block.
```

Comment:
Do you mind being more explicit? We expect the `IpcProvider` to be used by application. It may be useful to explain that this returns all the changes on the membership between two blocks. "validator change set" seems like an opaque concept.